pages/whats-left.markdown

```diff
@@ -0,0 +1,9 @@
+---
+layout: whats_left
+title:  "What's Left"
+date:   2021-03-29 01:45:00 +0900
+categories: numbers
+---
+
+<p>
+RustPython currently supports the full Python syntax. This is "what's left" from the Python Standard Library. For each module's details, check the <a href="https://docs.python.org/3/py-modindex.html">Python Module Index</a>. </p>
```

Comment:
I was blanking and struggling with my english. I used this as a placeholder text. Please update as you see fit.